src/components/Select/scss/index.scss

```diff
@@ -0,0 +1,103 @@
+@import "../../App/scss/variables.scss";
+
+.select {
+  position: relative;
+
+  &__title {
+    display: block;
+    position: relative;
+    cursor: pointer;
+    width: 100%;
+    border: 1px solid $blue;
+    box-shadow: 0 0 0 1px #fff;
+    input {
+      display: block;
+      width: 100%;
+      padding: 0.8rem 1rem;
+      font-size: 1rem;
+      outline: none;
+      border: none;
+      background: transparent;
+      &:active,
+      &:focus,
+      &::selection {
+        background: transparent;
+      }
+    }
+    &:after {
+      position: absolute;
+      display: block;
+      content: "";
+      right: 1rem;
+      top: 50%;
+      transform: translateY(-50%);
+      width: 0.5rem;
+      height: 0.5rem;
+      transition: all 0.3s ease;
+      background: url("../../../images/arrow.svg");
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+  }
+
+  &__options {
+    border-radius: 0;
+    box-shadow: 0 0 0 1px $blue;
+    border: 1px solid $blue;
+    background-color: $white;
+
+    position: absolute;
+    left: 0;
+    bottom: 0.6rem;
+    transform: translateY(100%);
+    width: 100%;
+    min-width: 80px;
+    min-height: 43px;
+    max-height: 40vh;
+    opacity: 0;
+    z-index: -1;
+    box-shadow: $select-menu-shadow;
+    overflow: hidden;
+    &--open {
+      opacity: 1;
+      z-index: 1;
+    }
+  }
+
+  &__option {
+    position: relative;
+    padding: $spacer $spacer + $spacer / 2;
+    &:not(:last-child)::after {
+      background-color: $gray;
+      bottom: 0;
+      content: "";
+      height: 1px;
+      left: $spacer;
+      position: absolute;
+      width: calc(100% - #{$spacer * 2});
+    }
+
+    &--disabled {
+      color: $gray-dark;
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: rgba(33, 18, 94, 0.1);
+    }
```

Comment:
enter (style nazi), please correct rest in the file and use such style further on.

Reply:
ok